### DISCUSSION TITLE
Does not work

### DISCUSSION THREAD

**Discussion by fizsef** [AdguardTeam/AdGuardHome#4073] [General]
I ask for help.
Version 107.2 OS version - OpenWRT 21.02.1 . Router Redmi AC1200.
The service is running, at http://192.168.0.1:3000 / the web interface opens.
![1](https://user-images.githubusercontent.com/74646605/147921546-7984112c-472e-4ce9-b39d-a5e031bb391d.jpg)

Files in place
![2](https://user-images.githubusercontent.com/74646605/147921545-a2fe6330-5877-44a7-a659-1322eae7a514.jpg)

But at the same time nothing works, it does not block.
Here is the contents of the AdGuard Home.yaml file
`bind_host: 192.168.0.1
bind_port: 3000
beta_bind_port: 0
users:
- name: root
  password: $2a$10$uh6Lf0QtJWwWD5ZdFB9vZ.QO19a9qHWnlFisykAoO5jV9J6bTxfLS
auth_attempts: 5
block_auth_min: 15
http_proxy: ""
language: ""
debug_pprof: false
web_session_ttl: 720
dns:
  bind_hosts:
  - 0.0.0.0
  port: 1745
  statistics_interval: 1
  querylog_enabled: true
  querylog_file_enabled: true
  querylog_interval: 2160h
  querylog_size_memory: 1000
  anonymize_client_ip: false
  protection_enabled: true
  blocking_mode: default
  blocking_ipv4: ""
  blocking_ipv6: ""
  blocked_response_ttl: 10
  parental_block_host: family-block.dns.adguard.com
  safebrowsing_block_host: standard-block.dns.adguard.com
  ratelimit: 20
  ratelimit_whitelist: []
  refuse_any: true
  upstream_dns:
  - 9.9.9.10
  - 149.112.112.10
  - 2620:fe::10
  - 2620:fe::fe:10
  upstream_dns_file: ""
  bootstrap_dns:
  - 9.9.9.10
  - 149.112.112.10
  - 2620:fe::10
  - 2620:fe::fe:10
  all_servers: false
  fastest_addr: false
  fastest_timeout: 1s
  allowed_clients: []
  disallowed_clients: []
  blocked_hosts:
  - version.bind
  - id.server
  - hostname.bind
  trusted_proxies:
  - 127.0.0.0/8
  - ::1/128
  cache_size: 4194304
  cache_ttl_min: 0
  cache_ttl_max: 0
  cache_optimistic: false
  bogus_nxdomain: []
  aaaa_disabled: false
  enable_dnssec: false
  edns_client_subnet: false
  max_goroutines: 300
  ipset: []
  filtering_enabled: true
  filters_update_interval: 24
  parental_enabled: false
  safesearch_enabled: false
  safebrowsing_enabled: false
  safebrowsing_cache_size: 1048576
  safesearch_cache_size: 1048576
  parental_cache_size: 1048576
  cache_time: 30
  rewrites: []
  blocked_services: []
  upstream_timeout: 10s
  local_domain_name: lan
  resolve_clients: true
  use_private_ptr_resolvers: true
  local_ptr_upstreams: []
tls:
  enabled: false
  server_name: ""
  force_https: false
  port_https: 443
  port_dns_over_tls: 853
  port_dns_over_quic: 784
  port_dnscrypt: 0
  dnscrypt_config_file: ""
  allow_unencrypted_doh: false
  strict_sni_check: false
  certificate_chain: ""
  private_key: ""
  certificate_path: ""
  private_key_path: ""
filters:
- enabled: true
  url: https://adguardteam.github.io/AdGuardSDNSFilter/Filters/filter.txt
  name: AdGuard DNS filter
  id: 1
- enabled: false
  url: https://adaway.org/hosts.txt
  name: AdAway Default Blocklist
  id: 2
whitelist_filters: []
user_rules: []
dhcp:
  enabled: false
  interface_name: ""
  dhcpv4:
    gateway_ip: ""
    subnet_mask: ""
    range_start: ""
    range_end: ""
    lease_duration: 86400
    icmp_timeout_msec: 1000
    options: []
  dhcpv6:
    range_start: ""
    lease_duration: 86400
    ra_slaac_only: false
    ra_allow_slaac: false
clients: []
log_compress: false
log_localtime: false
log_max_backups: 0
log_max_size: 100
log_max_age: 3
log_file: ""
verbose: false
os:
  group: ""
  user: ""
  rlimit_nofile: 0
schema_version: 12
`

**Comment by ainar-g** [comment]
Добрый день.  Вы [проставили адрес AdGuard Home в качестве DNS-сервера][1] на машинах клиентов?

[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/Getting-Started#configure-devices

**Reply by fizsef** [reply]
Добрый день. 
ДНС сервер у меня указан только в Wan интерфейсе. 
![1](https://user-images.githubusercontent.com/74646605/148088014-95577939-8b90-4fda-8c86-244ddf7e41ab.jpg)

А в  DHCP/DNS  ничего не указывал и не менял. 
![2](https://user-images.githubusercontent.com/74646605/148088003-b66b2eb4-c50c-4ab5-9d0e-b5d9582e90bb.jpg)

И при таких настройках на другом устройстве у меня все работает.

**Reply by ainar-g** [reply]
Прошу прощения за долгий ответ.  Вы пробовали проставить адрес AGH в оба поля в настройках?  А также обращаться к AGH напрямую через `nslookup`?
